Gems/Atom/Feature/Common/Assets/Materials/Types/NewPBR/NewPBR_Lighting.azsli

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <Atom/Features/PBR/Lights/Lights.azsli>
+#include <Atom/Features/PBR/Lights/Ibl.azsli>
+#include <Atom/Features/PBR/Decals.azsli>
+
+LightingData EvaluateLighting(Surface surface, float4 screenPosition)
```

Comment:
I suggest calling the file EvaluateLighting.azsli to make it clear that it provides this specific function.